static/app/components/events/eventDataSection.tsx

```diff
@@ -128,6 +130,14 @@ const Permalink = styled('a')`
   }
 `;
 
+const PermalinkAnchor = styled(Anchor)`
```

Comment:
Use `<Anchor />` to get a better focus state.

Before:
<img width="954" alt="Screen Shot 2022-05-02 at 9 48 45 AM" src="https://user-images.githubusercontent.com/44172267/166289861-b5b42cea-e69c-4733-a021-55b19172a496.png">

After:
<img width="953" alt="Screen Shot 2022-05-02 at 9 49 23 AM" src="https://user-images.githubusercontent.com/44172267/166289952-a4ae6172-afcc-4d5e-9ed6-0e848dae72ce.png">